docs/spec/spec.md

```diff
@@ -176,6 +180,20 @@ The conforming implementation of the specification is released and included in t
             * 10.5.1.1 [The `@subgraph:Subgraph` Annotation](#10511-the-subgraphsubgraph-annotation)
             * 10.5.1.2 [The `@subgraph:Entity` Annotation](#10512-the-subgraphentity-annotation)
             * 10.5.1.3 [The `subgraph:ReferenceResolver` Function Type](#10513-the-subgraphreferenceresolver-function-type)
+11. [Experimental Features](#11-experimental-features)
+    * 11.1 [DataLoader](#111-dataloader)
+        * 11.1.1 [DataLoader API](#1111-dataloader-api)
+            * 11.1.1.1 [The `load` method](#11111-the-load-method)
+            * 11.1.1.2 [The `get` method](#11112-the-get-method)
+            * 11.1.1.3 [The `dispatch` method](#11113-the-dispatch-method)
+            * 11.1.1.4 [The `clearAll` method](#11114-the-clearall-method)
+        * 11.1.2 [The DefaultDataLoader](#1112-the-defaultdataloader)
+            * 11.1.2.1 [The `init` Method](#11121-the-init-method)
+                * 11.1.2.1.1 [The BatchLoadFunction](#111211-the-batchloadfunction)
```

Comment:
Shouldn't the batchload function be in a separate section? Maybe under the `DataLoader API` section? The `init` method too is part of the DataLoader API, right?

Reply:
As discussed offline, since we can't have an init method in a Ballerina (DataLoader) object, we'll keep this as is